jenkins/docker/docker-re-release.jenkinsfile

```diff
@@ -0,0 +1,59 @@
+lib = library(identifier: 'jenkins@5.7.1', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/opensearch-project/opensearch-build-libraries.git',
+]))
+
+pipeline {
+    options {
+        timeout(time: 7, unit: 'HOURS')
+    }
+    agent none
+    parameters {
+        string(
+            name: 'PRODUCT',
+            description: "OpenSearch / OpenSearch-Dashboards",
+            trim: true
+        )
+        string(
+            name: 'TAG',
+            description: "Enter tag of the Product",
+            trim: true
+        )
+        booleanParam(
+            name: 'RE_RELEASE',
+            defaultValue: true,
+            description: "Forces to create a Docker-Copy if the Docker Build is successful"
```

Comment:
This description is not clear, can you please elaborate or be more clear? 
Thanks!

Reply:
Agree ^^